src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,107 @@
+package duke;
+
+import java.util.List;
+
+/**
+ * Control all jobs related to the User Interface
+ * the class has methods to greet users & print feedback for users' input
+ */
+public class Ui {
+    protected static String separatedLine;
+
+    /**
+     * print greeting content
+     */
+    public static void greet() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println(separatedLine);
```

Comment:
This line of code is reused many times within this class. Maybe you might want to put it into it's own function to reuse, like `sendLine()` or something similar?